src/data/cookies.ts

```diff
@@ -2,10 +2,11 @@ import type { UserRole } from "@type/roles";
 
 const mockCookies = {
     username: "John Doe",
-    id: "1",
-    role: "Applicant" as UserRole //'Applicant' | 'Authorizer' | 'Admin' | 'AccountsPayable' | 'TravelAgency';
+    id: "3",
+    department_id: "3",
+    role: "AccountsPayable" as UserRole //'Applicant' | 'Authorizer' | 'Admin' | 'AccountsPayable' | 'TravelAgency';
```

Comment:
So that not everyone is conflicted and the system keeps changing from time to time, it is better to leave it the same for now.

```suggestion
    role: "Applicant" as UserRole //'Applicant' | 'Authorizer' | 'Admin' | 'AccountsPayable' | 'TravelAgency';
```